presto-hive-metastore/src/main/java/com/facebook/presto/hive/metastore/SemiTransactionalHiveMetastore.java

```diff
@@ -505,8 +509,8 @@ public synchronized void dropColumn(MetastoreContext metastoreContext, String da
 
     private ConnectorCommitHandle buildCommitHandle(SchemaTableName table, MetastoreOperationResult operationStats)
     {
-        Map<SchemaTableName, List<DateTime>> lastDataCommitTimes = ImmutableMap.of(table, operationStats.getLastDataCommitTimes());
-        return new HiveCommitHandle(lastDataCommitTimes);
+        Map<SchemaTableName, List<Integer>> lastDataCommitTimes = ImmutableMap.of(table, operationStats.getLastDataCommitTimes());
```

Comment:
It's safer and more future proof to store timestamps as longs rather than ints (here and throughout).

Reply:
Sounds good. let me update all of them!

Reply:
Done